biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -1131,23 +1159,30 @@ subroutine LeafLayerPhotosynthesis(f_sun_lsl,         &  ! in
                   cquad = stomatal_intercept_btran*stomatal_intercept_btran + &
                           (2.0*stomatal_intercept_btran + term * &
                           (1.0 - medlyn_slope(ft)* medlyn_slope(ft) / vpd)) * term
-           
+
                   call quadratic_f (aquad, bquad, cquad, r1, r2)
                   gs_mol = max(r1,r2)
 
-                else if ( stomatal_model == 1 ) then         !stomatal conductance calculated from Ball et al. (1987)
+               else if ( stomatal_model == 1 ) then         !stomatal conductance calculated from Ball et al. (1987)
+                  if (use_agross == 1) then
```

Comment:
I can't find where use_agross is defined.  
Should we modify this logic anyway to just use agross if LWP is provided?

Reply:
Notes taken with @JunyanDing during zoom chat:
```
logical, parameter :: use_agross = .false.

if (use_agross .or. (  (hlm_use_planthydro.eq.itrue).and.     (EDPftvarcon_inst%hydr_k_lwp(ft)>0.))) then
```